experimental-default-global-styles.json

```diff
@@ -1,7 +1,41 @@
 {
-	"color": {
-		"primary": "#52accc",
-		"background": "white",
-		"text": "black"
+	"global": {
+		"presets": {
+			"color": {
+				"black": "#000000",
```

Comment:
Should we include the name property, even though it is not yet translatable so we have the final shape we are going to work with e.g:
```
color: [ {
	slug: 'black',
	name: 'Black',
	color: '#abb8c3'
}, ... ]
```

Reply:
I'm thinking that can be done later when those things are pulled from this file. At the moment, I'd like to keep this minimal.

Reply:
I second @jorgefilipecosta even if we don't include the name now, we should use an array as the order is important.

Reply:
Ah, exposing the final format (although not the name property) is a good call. It prevents theme authors from needing to change it later. Done at https://github.com/WordPress/gutenberg/pull/20290/commits/e484d7b7080e7b539b497a0a5a9e59ed45f45315